src/model/localSave.js

```diff
@@ -0,0 +1,30 @@
+import Project from '../controller/project';
+import Todo from '../controller/todo';
+
+let localProjectList = []; // eslint-disable-line import/no-mutable-exports
+
+function populateProjects() {
```

Comment:
You ought to use ES6 Syntax for this project:
Which means usage of arrow functions instead of the function keyword.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from pathwright